CMakeLists.txt

```diff
@@ -47,6 +47,13 @@ SET(CMAKE_ALLOW_LOOSE_LOOP_CONSTRUCTS true)
 SET(CMAKE_CXX_STANDARD 11)
 SET(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
+  # require at least gcc 4.9 due to bug https://gcc.gnu.org/bugzilla/show_bug.cgi?id=41933
+  if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 4.9)
+    message(FATAL_ERROR "GCC version must be at least 4.9!")
```

Comment:
@jpfeuffer @timosachsenberg this is actually a problem for many systems, including the manylinux build system that relies on gcc 4.8 for compilation. Is there any way to revert this to 4.8 again?

Reply:
I think it would require working around the gcc bug.
So I read a bit into this and it seems that more and more projects are moving away from pip/manywheels to conda as it doesn't have these kind of restrictions.
I sounds to me that this might need to happen in the future but would like to hear your opinion on that.

Reply:
I am happy to move away from `manylinux` at some point, the question is how does the work to port the build infrastructure to conda?